src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const male = people.filter(man => man.sex === 'm');
+  let finalMale;
+
+  century
+    ? finalMale = male.filter(man => Math.ceil(man.died / 100) === century)
+    : finalMale = male;
```

Comment:
```suggestion
const finalMale = people.filter(man =>
    century
      ? Math.ceil(man.died / 100) === century && man.sex === 'm'
      : (man.sex === 'm')
  );
```

You can simplify your code like this using only one .filter method instead of two.